### ISSUE TITLE
a more radical resize rate?

### ISSUE THREAD

**Issue by stupiding** [tianshuichen/Wavelet-like-Auto-Encoder#3]
Have you ever tried a more radical resize rate, such as 76x76 or 56x56?

**Comment by tianshuichen** [comment]
Yes, we have tried resizing the input image into two quarter-resolution sub-images, but the results are not satisfactory. We are now exploring these.
